lapce-core/src/indent.rs

```diff
@@ -32,21 +33,10 @@ impl IndentStyle {
     pub fn as_str(&self) -> &'static str {
         match *self {
             IndentStyle::Tabs => "\t",
-            IndentStyle::Spaces(1) => " ",
-            IndentStyle::Spaces(2) => "  ",
-            IndentStyle::Spaces(3) => "   ",
-            IndentStyle::Spaces(4) => "    ",
-            IndentStyle::Spaces(5) => "     ",
-            IndentStyle::Spaces(6) => "      ",
-            IndentStyle::Spaces(7) => "       ",
-            IndentStyle::Spaces(8) => "        ",
-
+            IndentStyle::Spaces(x) if x <= Self::LONGEST_INDENT.len() as u8 => Self::LONGEST_INDENT.split_at(x.into()).0,
             // Unsupported indentation style.  This should never happen,
-            // but just in case fall back to two spaces.
-            IndentStyle::Spaces(n) => {
-                debug_assert!(n > 0 && n <= 8); // Always triggers. `debug_panic!()` wanted.
-                "  "
-            }
+            // but just in case fall back to the default of 4 spaces
+            _ => "    "
```

Comment:
Any reason why this was changed? Panicking in debug builds is a good way to catch unintended behaviour. Also, since this branch is supposed to be unreachable, why change from 2 spaces to 4?